### PR TITLE
[13.0][MIG] l10n_nl_bsn

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

**Review by lfreeke — APPROVED** [review]
Functional test

File: l10n_nl_bsn/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2016-2019 Onestein (<https://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Burgerservicenummer (BSN) for Partners',
+    'version': '13.0.1.0.1',
```
Comment (StephanRozendaal) [inline l10n_nl_bsn/__manifest__.py]: Shouldn't the version number start at `13.0.1.0.0` again?

Comment (astirpe) [reply l10n_nl_bsn/__manifest__.py]: Right! :+1:

**Review by StephanRozendaal — APPROVED** [review]

**Pull request merged**